### ISSUE TITLE
HammerRegistry getRewards method not found

### ISSUE THREAD

**Issue by bmhimes** [jaredlll08/ModTweaker2#276]
The below error:
```
java.lang.NoSuchMethodError: exnihilo.registries.HammerRegistry.getRewards()Ljava/util/HashMap;
	at modtweaker2.mods.exnihilo.handlers.Hammer$Add.<init>(Hammer.java:79)
	at modtweaker2.mods.exnihilo.handlers.Hammer.addRecipe(Hammer.java:74)
	at modtweaker2.mods.exnihilo.handlers.Hammer.addRecipe(Hammer.java:44)
	at Wyld.__script__(wyld.zs:11)
	at __ZenMain__.run(wyld.zs)
	at minetweaker.runtime.MTTweaker.load(MTTweaker.java:163)
	at minetweaker.MineTweakerImplementationAPI.reload(MineTweakerImplementationAPI.java:651)
	at minetweaker.MineTweakerImplementationAPI$1.execute(MineTweakerImplementationAPI.java:82)
	at minetweaker.MineTweakerImplementationAPI$19.execute(MineTweakerImplementationAPI.java:637)
	at minetweaker.mc1710.server.MCServer$MCCommand.func_71515_b(MCServer.java:124)
	at net.minecraft.command.CommandHandler.func_71556_a(CommandHandler.java:94)
	at net.minecraft.network.NetHandlerPlayServer.func_147361_d(NetHandlerPlayServer.java:739)
	at net.minecraft.network.NetHandlerPlayServer.func_147354_a(NetHandlerPlayServer.java:718)
	at net.minecraft.network.play.client.C01PacketChatMessage.func_148833_a(SourceFile:37)
	at net.minecraft.network.play.client.C01PacketChatMessage.func_148833_a(SourceFile:9)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
```
is produced by the following example code from the wiki:
```
mods.exnihilo.Hammer.addRecipe(<minecraft:cactus>, <minecraft:dye:2>, 0.45, 1.25);
```
at [ModTweaker:Ex Nihilo Support](http://minetweaker3.powerofbytes.com/wiki/ModTweaker:Ex_Nihilo_Support).

**Environment:**
MineTweaker version: MineTweaker3-1.7.10-3.0.10
ModTweaker version: ModTweaker2-0.9.4
Base mod pack: SkyFactory 2.4

**Comment by jaredlll08** [comment]
Could you please provide me with the ex-nihilo version?

**Comment by bmhimes** [comment]
Ex Nihilo is 1.38-36, as included with SkyFactory 2.4.

**Comment by jaredlll08** [comment]
Did you happen to update modtweaker? if not, the pack is using an outdated ex-nihilo.
You can find the latest here: http://ci.tterrag.com/job/Ex-Nihilo/
